### PR TITLE
Story 3 Task 5 

### PR DESCRIPTION
**Pull request by CMeier556**
Email successfully sends, console.log will let you know that after you input data. New email box closes after email is sent and sorted by the API

### CODE REVIEW

File: src/Components/NewEmail/index.jsx [hunk 1]
```diff
@@ -1,27 +1,66 @@
...
+    }
+
+    const handleBody = (e) => {
+        setBody(e.target.value)
+    }
+    
+    const sendEmail = async () => {
+        const sentEmail = {
+            email: address,
+            name: 'Alex is a loser',
```
Comment (alexjames4) [inline src/Components/NewEmail/index.jsx:25]: hmm

File: src/Components/NewEmail/index.jsx [hunk 2]
```diff
@@ -1,27 +1,66 @@
...
+            body: body
+        }
+        const response = await fetch (`http://localhost:8080/emails`, {
+            method: 'POST',
+            body: JSON.stringify(sentEmail),
+            headers: {
+                'content-type': 'application/json'
+            }
+        })
+        let emailResponseData = await response.json()
```
Comment (alexjames4) [inline src/Components/NewEmail/index.jsx:36]: const

File: src/Components/NewEmail/index.jsx [hunk 3]
```diff
@@ -1,27 +1,66 @@
...
+        if (emailResponseData.data.sent) {
+            cancelNewEmail()
```
Comment (alexjames4) [inline src/Components/NewEmail/index.jsx:38]: love it

File: src/Components/NewEmail/index.jsx (hunk 1 above)
Comment (CMeier556) [reply src/Components/NewEmail/index.jsx:25]: No

**Pull request merged**